src/app/renderer.js

```diff
@@ -34,13 +34,14 @@ function Renderer (editor, updateFiles, udapp, executionContext, formalVerificat
   })
 }
 
-Renderer.prototype.error = function (message, container, noAnnotations, type) {
+Renderer.prototype.error = function (message, container, options) {
   var self = this
-  if (!type) {
-    type = utils.errortype(message)
+  var opt = options || {}
+  if (!opt.type) {
+    opt.type = utils.errortype(message)
   }
-  var $pre = $('<pre />').text(message)
-  var $error = $('<div class="sol ' + type + '"><div class="close"><i class="fa fa-close"></i></div></div>').prepend($pre)
+  var $pre = $(opt.useSpan ? '<span />' : '<pre />').text(message)
```

Comment:
The pre was mostly needed to format the pointer to the source location properly, which is generated by the compiler. If we use `span` here, the browser breaks the lines and it is much easier to read.